packages/web-react/src/components/TextArea/README.md

```diff
@@ -10,20 +10,20 @@ and show if the textarea is required.
 
 ## Available props
 
-| Name              | Type                          | Description                       |
-| ----------------- | ----------------------------- | --------------------------------- |
-| `id`              | string                        | Textarea and label identification |
-| `name`            | string                        | Textarea name                     |
-| `label`           | string                        | Label text                        |
-| `placeholder`     | string                        | Textarea placeholder              |
-| `value`           | string                        | Textarea value                    |
-| `maxLength`       | number                        | Maximum number of characters      |
-| `message`         | string                        | Validation or help message        |
-| `rows`            | number                        | Number of visible rows            |
-| `validationState` | `success`, `warning`, `error` | Type of validation state          |
-| `isDisabled`      | boolean                       | Whether is field disabled         |
-| `isRequired`      | boolean                       | Whether is field required         |
-| `isLabelHidden`   | boolean                       | Whether is label hidden           |
+| Name              | Type                                    | Description                       |
+| ----------------- | --------------------------------------- | --------------------------------- |
+| `id`              | string                                  | Textarea and label identification |
+| `name`            | string                                  | Textarea name                     |
+| `label`           | string                                  | Label text                        |
+| `placeholder`     | string                                  | Textarea placeholder              |
+| `value`           | string                                  | Textarea value                    |
+| `maxLength`       | number                                  | Maximum number of characters      |
+| `message`         | string                                  | Validation or help message        |
+| `rows`            | number                                  | Number of visible rows            |
+| `validationState` | `success`, `warning`, `danger`, `error` | Type of validation state          |
```

Comment:
link to docs/dictionaries.md and deprecation info please

Reply:
done